crawler.py

```diff
@@ -756,6 +766,23 @@ def save(self, data, name='results.json'):
                 data, f, indent=2, sort_keys=True, separators=(',', ': '))
         self.logger.info("Saved data to %s", name)
 
+    def handle_clicking(self, domain):
+        # fetch the clickable elements on given domain
+        clickables = self.driver.find_elements_by_tag_name('a') + self.driver.find_elements_by_tag_name('button')
```

Comment:
Buttons don't have an href, so they never get matched, so there is no reason to select them.